sdk/provisioning/Azure.Provisioning/src/Expressions/BicepStringBuilder.cs

```diff
@@ -102,10 +102,11 @@ public void AppendFormatted<T>(T t)
         }
         else
         {
-            _expressions.Add(
-                t is null ?
-                    BicepSyntax.Null() :
```

Comment:
If you wanted to interpolate `${null}` into the string, you would use `BicepSyntax.Null()` correct?

```C#
.Append($".blob.core.windows.net{BicepSyntax.Null()}");
```

Reply:
Correct - though you'd need `new NullLiteralExpression()` because `BicepSyntax` is internal.